src/libraries/System.Private.CoreLib/src/System/Threading/Timer.cs

```diff
@@ -881,6 +881,14 @@ public Timer(TimerCallback callback)
             TimerSetup(callback, this, DueTime, Period);
         }
 
+        public Timer(TimeSpan interval)
```

Comment:
This ctor doesn't make sense and should be deleted.

Reply:
If the constructor is meaningless, why is it API-approved?

Reply:
It's not.  This is System.Threading.Timer.  The `Timer` for which a ctor was discussed in the API issue is System.Timers.Timer... that's not the same thing.

Reply:
Oh yeah, I didn't pay attention to that. I will remove that.